assets/src/styles/layouts/_reading.scss

```diff
@@ -30,7 +30,7 @@ h1.reading-header__title {
   #content {
     font-size: rem(18);
     flex-grow: 1;
-    padding: 0 1rem;
+    padding: 3rem 1rem 0;
```

Comment:
Restore 3rem padding at top of content block to give content same spacing as previously provided by reading-header